### PR TITLE
Simplify autoresponder logic

### PR DESCRIPTION
**Pull request by thecutefinder**
There is no longer a dedicated Chinese support channel, so it's unnecessary to check if a thread is in the Chinese support channel.

### CODE REVIEW

**Review by memetrollsXD — CHANGES_REQUESTED** [review]

File: src/events/messageCreate.ts [hunk 1]
```diff
@@ -43,7 +38,7 @@ function filterInvis(content: string) {
 export default async function run(message: Message) {
     if (message.author.bot) return;
 
-    if (message.channelId === "1028327705571238009" && message.deletable) { // #verify
+    if (message.channelId === "1082829744363548762" && message.deletable) { // #verify
```
[the remark this answers is not in the document]
Comment (thecutefinder) [reply src/events/messageCreate.ts:41]: Accidentally changed that to the wrong one in testing

File: src/events/messageCreate.ts [hunk 2]
```diff
@@ -55,21 +50,14 @@ export default async function run(message: Message) {
...
             if (action) {
-                switch (message.channel.id) {
-                    case $support_cn:
-                        if (action.action_cn) message.reply(action.action_cn);
-                        break;
-                    default:
-                        if (action.action) message.reply(action.action);
-                        break;
-                }
+                message.reply(action.action);
```
[the remark this answers is not in the document]
Comment (thecutefinder) [reply src/events/messageCreate.ts:60]: Yes, but we would have to upgrade to a newer Discord.JS version, which would require re-writing a lot of the bot's code, due to the newer versions changing class and variable names.

[the remark this answers is not in the document]
Comment (thecutefinder) [reply src/events/messageCreate.ts:60]: I can work on that later today.

**Review by memetrollsXD — APPROVED** [review]
Channel ID is proper now, although the Chinese reply is gone without any replacement, I haven't seen a lot of Chinese support posts anyway, so this will do for now since it's only a utility

**Comment by thecutefinder** [comment]
@memetrollsXD Just so you know, I've looked into trying to check the tags to add a CN reply and check the post title to match keywords based on that as well.

Post title is possible using the `threadCreate` event for each new post in #support. Given that the bot will add a reaction once it's responded, you can check the reaction count to prevent race conditions in which a keyword has been matched in both the contents and title. This could be made more efficient and work better for cases where the same keyword has duplicated by checking replies in the thread and making sure that the same response hasn't already been sent, but I'm a bit ~~lazy~~ busy right now and don't want to look into it.

As for the tags: This is what confuses me the most. Okay, so, when using the `messageCreate` event, you cannot check what thread the post is inside of. You CAN check what forum it's inside of, but that's the most. Therefore, unless we do something really complicated<sup>1</sup>, the tags can only be checked in the `threadCreate` event using a tags attribute (and therefore we can only switch between CN and EN responses in replies to the initial forum post, not in subsequent messages). HOWEVER, it does not return tag names or any objects, only their IDs, and I have *no idea* how to get tag IDs. I have developer mode on and everything in Discord, but I simply *cannot* find any way to initiate a dropdown or copy the IDs for any tags. I looked everywhere that tags are listed and I cannot find anything. I'll probably have to open DevTools and find it manually or something.

<sup>1</sup> You would have to check all open in threads in some way (I think there is some way to fetch them all at once), and then in `messageCreate`, find where new messages match up exactly with something stored in an open thread. Using this, you could check the thread metadata and such, including the IDs of any tags, and use this to choose to either write a CN or EN comment. However, this is very, very complicated, and I don't want to do it right now.

**Pull request merged**